project/LinkerdBuild.scala

```diff
@@ -12,7 +12,13 @@ object LinkerdBuild extends Base {
   val Bundle = config("bundle") extend Minimal
   val Dcos = config("dcos") extend Bundle
 
+  val configCore = projectDir("config")
+    .withTwitterLibs(Deps.finagle("core"))
+    .withLibs(Deps.jackson)
+    .withLib(Deps.jacksonYaml)
+
   val consul = projectDir("consul")
+    .dependsOn(configCore)
```

Comment:
I don't _love_ that consul is picking up extra deps, but its probably the least worst option for the time being.